### PR TITLE
Windows fix cmd

### PR DESCRIPTION
**Pull request by colorgreen**
Because previous pull request was closed I had to open this one. I did tests on Windows and Debian, both passed. Removed error of TTY allocation. Please test it last time, I still think it is good solution @KABBOUCHI

### CODE REVIEW

**Comment by taylorotwell** [comment]
Someone will have to confirm this.

File: src/RemoteProcessor.php [hunk 1]
```diff
@@ -45,13 +45,24 @@ protected function getProcess($host, Task $task)
...
 
-            $process = new Process(
-                "ssh $target 'bash -se' << \\$delimiter".PHP_EOL
+            if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+
+                $process = new Process("ssh $target -T" );
+
+                $process->setInput(implode(PHP_EOL, $env));
+                $process->setInput('set -e');
+                $process->setInput( str_replace( "\r", '', $task->script ));
```
[the remark this answers is not in the document]
Comment (KABBOUCHI) [reply src/RemoteProcessor.php]:
```php
$process->setInput(
                    implode(PHP_EOL, $env)
                    .'set -e ' . PHP_EOL
                    .str_replace( "\r", '', $task->script )
                );
```

[the remark this answers is not in the document]
Comment (colorgreen) [reply src/RemoteProcessor.php]: yea, you're right

**Comment by KABBOUCHI** [comment]
@taylorotwell It's working for me
screenshot_2

**Pull request merged**